src/server/serve.rs

```diff
@@ -258,6 +259,35 @@ impl InnerService {
         }
     }
 
+    fn get_content_encoding<'a>(
+        &'a self,
+        req: &'a Request,
+        res: &'a mut Response,
+        mime_type: &'a mime::Mime,
+    ) -> Option<Cow<'a, str>> {
```

Comment:
I see a `Cow` here. Any reason hinders us from using a plain `&str`?

Reply:
Since `br`, `deflate`, `gzip` and `identity` are `&'static str`, I return `&'static str` on c616445 instead.